src/components/select/index.md

```diff
@@ -39,4 +39,16 @@ You can add hint text to help the user understand the options and choose one of
 
 ## Research on this component
 
-Watch a video about how some [users struggle with selects](https://www.youtube.com/watch?v=CUkMCQR4TpY).
+User research has shown that some users struggle with selects.
+
+### Known issues and gaps
+
+Research shows that users can struggle with selects, particularly when users have:
+  - been unable to close the select
```

Comment:
one of the main issues I'm aware of is people not knowing that they can scroll the select or how to, worth adding?

Reply:
I'll add that in